### DISCUSSION TITLE
how to get "loading" or "searching" status from const {query, refine} = useSearchBox({});

### DISCUSSION THREAD

**Discussion by alainib** [algolia/instantsearch#5699] [Q&A]
i get a code where `const {query, refine} = useSearchBox({});` is used.

i need a boolean to know when the `refine` query is done. 

how to do it please ?

**Comment by Haroenv** [comment]
You can use the `status` from useInstantSearch which can be `idle`, `loading`, `stalled` or `error`. There's however no link between refine and its direct effect, the status is global.
